tests/desktop/test_users_account.py

```diff
@@ -86,13 +86,21 @@ def test_hide_email_checkbox_works(self, mozwebqa):
         initial_state = view_profile_page.is_email_field_present
 
         edit_profile_page = home_page.header.click_edit_profile()
-        edit_profile_page.change_hide_email_state()
-        edit_profile_page.click_update_account()
+        # make sure email checkbox isn't already selected
```

Comment:
With the above changes made to the page object, you shouldn't need any of this code. If we expect the box to be initially unchecked (and I think that's true, but I could be wrong), then all you need to do is call `edit_profile_page.hide_email_address()` and it will take care of everything for you.